### ISSUE TITLE
Questions about the competing method in Figure 5 (c)

### ISSUE THREAD

**Issue by ChenhongyiYang** [yuantn/MI-AOD#70]
Hello,

Thank you for the fantastic work. May I ask a question about Figure 5 (c)? I notice the Core-set is listed as a competing method on COCO, which is an important result, and I believe this is the first paper that presents it. 

In fact, I am trying to realize the same result on COCO. However, I find that to get the result I need to infer the model on the whole COCO training set (~ 118k images) and compute the distance for every pair of images, which is memory and time consuming: the distance matrix size is nearly 40GB. Moreover, running mixed integer programming (MIP) is even more expensive. 

Therefore, may I ask how did you work this out? Did you run the full Core-set algorithm that contains the MIP? Or only the Greedy K-Centroid was used?

Additionally, could you please provide the initial COCO training set (the 2% one)? As active learning is prone to variation in initialization. Starting from the same initialization can make sure the following works to have a fair comparison with yours. 

Looking forward to your answer :-)

**Comment by yuantn** [comment]
Hello,

Thanks for your attention to our work.

For Core-set, we use the function [in this repository](https://github.com/dsgissin/DiscriminativeActiveLearning/blob/master/query_methods.py#L487) for implementation. So we only use the Greedy K-Centroid method.

For COCO training set, we set the random seed [here](../blob/master/tools/train.py#L104) before initialization. You may use the same random seed 666.

Hope this answer would be useful to you :-)

**Comment by ChenhongyiYang** [comment]
Cool!. Thank you for the quick reply. It's very helpful. :-)

**Comment by ChenhongyiYang** [comment]
Hi. Sorry for bothering again. May I ask how many epochs did you train for the competing methods that did not require adversarial training? Is it still 26 epochs?

**Comment by yuantn** [comment]
Yes.

**Comment by ChenhongyiYang** [comment]
Hi. But I found that a simple ResNet50 RetinaNet model with 26 epoch training can achieve ~14 AP using your initial set (2% images), which is far better than the numbers reported in the paper. Is there anything wrong?

**Comment by ChenhongyiYang** [comment]
Oh sorry, just found a bug in my code.... My bad :-(
